### PR TITLE
TC07

### PR DESCRIPTION
**Pull request by trangnhomanhcoi2601**

### CODE REVIEW

File: src/main/java/page_objects/BasePage.java [hunk 1]
```diff
@@ -13,32 +13,65 @@
     private final By tabMyTicket = By.cssSelector("li a[href='/Page/ManageTicket.cshtml']");
     private final By tabChangePassword = By.cssSelector("li a[href='/Account/ChangePassword.cshtml']");
     private final By tabLogout = By.cssSelector("li a[href='/Account/Logout']");
+    private final By tabRegister = By.cssSelector("li a[href='/Account/Register.cshtml']");
+
```
Comment (anhlvt) [inline src/main/java/page_objects/BasePage.java:17]: dư

File: src/test/java/tests/RegisterTests.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import helpers.LogHelper;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import page_objects.RegisterPage;
+
+public class RegisterTests extends BaseTests {
+
+    RegisterPage registerPage = new RegisterPage();
+    private final String email = DataHelper.getRandomEmail();
+    private final String password = DataHelper.getRandomPassword();
```
Comment (tanmle) [inline src/test/java/tests/RegisterTests.java]: you don't need to random a password

File: src/test/java/tests/RegisterTests.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    private final String pid = DataHelper.getRandomNumber();
```
Comment (anhlvt) [inline src/test/java/tests/RegisterTests.java]: no need to random password

File: src/main/java/page_objects/BasePage.java (hunk 1 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/BasePage.java:17]: I removed it

File: src/test/java/tests/RegisterTests.java (hunk 3 above)
Comment (trangnhomanhcoi2601) [reply src/test/java/tests/RegisterTests.java]: I removed it successfully. Help me check it again

File: src/main/java/page_objects/RegisterPage.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        return DriverHelper.getDriver().findElement(txtPid);
+    }
+
+    private WebElement getBtnRegister() {
+        return DriverHelper.getDriver().findElement(btnRegister);
+    }
+
+    //Methods
+    public String getSuccessMessage() {
+        return DriverHelper.getDriver().findElement(lblSuccessMessage).getText();
```
Comment (MinhQuan119) [inline src/main/java/page_objects/RegisterPage.java]: Create getLblSuccessMessage method to find it first, then using it in this method

**Review by tanmle — APPROVED** [review]

**Review by tanmle — APPROVED** [review]

File: src/main/java/page_objects/RegisterPage.java (hunk 4 above)
Comment (trangnhomanhcoi2601) [reply src/main/java/page_objects/RegisterPage.java]: I created getLblSuccessMessage method. Help me check it, thanks so much

**Review by MinhQuan119 — APPROVED** [review]

**Pull request merged**